src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -927,6 +927,17 @@ public BoundStatement ThrowNull()
             return Throw(Null(Compilation.GetWellKnownType(Microsoft.CodeAnalysis.WellKnownType.System_Exception)));
         }
 
+        public BoundExpression ThrowNullExpression(TypeSymbol type)
```

Comment:
It was used in the translation of the match expression. Since we're not pulling that in now, I'll remove this helper.

---
In reply to: [76682899](https://github.com/dotnet/roslyn/pull/13437#discussion_r76682899) [](ancestors = 76682899)